app/commands/v2/publish.rb

```diff
@@ -93,7 +93,7 @@ def publish_content_item(draft_content_item)
       def build_live_attributes(draft_content_item)
         attributes = draft_content_item
           .attributes
-          .except("access_limited", "version")
+          .except("version")
```

Comment:
Given we still have the access limited column on draft content item, should we still strip this field out?